.github/workflows/docs.yml

```
@@ -15,6 +15,11 @@ jobs:
         with:
           node-version-file: .nvmrc
           cache: npm
+      - uses: opensea-public-docs-publisher
+        id: get_token
+        with:
+          app_id: 409633
```

please use the secret here
```suggestion
        app_id: ${{ secrets.PUBLIC_DOC_PUBLISHER_APP_ID }}
```

It's not really a secret, but makes it so if we change to a new app it doesn't need workflow changes